guides/game_development/animations_from_system_calls.md

```diff
@@ -1,16 +1,12 @@
----
-order: 100
----
+# Animations from system calls
```

Comment:
Simplified the title, updated the file path (url slug) to match, and changed ~most things to use sentence case rather than capitalized headings